torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2277,12 +2300,23 @@ def _produce_dyn_sizes_from_int_tuple(self,
             ))
         return size
 
+    # TODO: This is just for testing convenienced, rename accordingly
     def create_symbolic_sizes_strides_storage_offset(
         self,
         ex: torch.Tensor,
         source: Source,
         *,
         symbolic_context: Optional[SymbolicContext] = None,
+    ):
+        inp = CreateSymbolicSizesStridesStorageOffsetInput.from_tensor(ex)
+        return self.hermetic_create_symbolic_sizes_strides_storage_offset(inp, source, symbolic_context=symbolic_context)
+
+    def hermetic_create_symbolic_sizes_strides_storage_offset(
```

Comment:
I find hermetic here to be a little bit confusing. would `create_symbolic_sizes_strides_storage_offset_from_tensor` method above renaming, and `create_symbolic_sizes_strides_storage_offset` naming here make more sense ?


There's not reallyyyy anything more hermetic from this method and the other. They both do the same thing using the ex input to get the required metadata